tests/drivers/mm/sys_mm_drv_api/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  drivers.mm.sys_mm_drv_api.intel_adsp:
+    tags: drivers
+    platform_allow: intel_adsp_cavs15 intel_adsp_cavs18 intel_adsp_cavs20 intel_adsp_cavs25
```

Comment:
why those only? Looks like a generic api

Reply:
You name it, muscle memory, copypasta, etc =P
You're right, no need to restrict testing at all.